test/test_run_analysis.rb

```diff
@@ -50,6 +50,88 @@ def _assert_and_puts(output, msg, expect_error = true)
+      next if _expected_warning_message(message, "Both 'cooking_range' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'hot_water_fixtures' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'hot_water_fixtures' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'hot_water_fixtures' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'plug_loads_other' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'plug_loads_other' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'plug_loads_other' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and weekday fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and weekend fractions provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, 'Could not find state average propane rate based on')
+      next if _expected_warning_message(message, 'Could not find state average fuel oil rate based on')
+      next if _expected_warning_message(message, "Specified incompatible corridor; setting corridor position to 'Single Exterior (Front)'.")
+      next if _expected_warning_message(message, 'DistanceToTopOfWindow is greater than 12 feet; this may indicate incorrect units. [context: /HPXML/Building/BuildingDetails/Enclosure/Windows/Window/Overhangs[number(Depth) > 0]')
+      next if _expected_warning_message(message, 'Home with conditioned basement has floor insulation.')
+      next if _expected_warning_message(message, 'Ducts are entirely within conditioned space but there is moderate leakage to the outside. Leakage to the outside is typically zero or near-zero in these situations, consider revising leakage values. Leakage will be modeled as heat lost to the ambient environment.') # FIXME: remove once duct TSVs are refactored
+      next if _expected_warning_message(message, 'No emissions factor found for Scenario=LRMER_MidCase_15, Type=CO2e, Fuel=Natural Gas.') # FIXME: remove once OS-HPXML has_fuel method is fixed
+
+      if !testing
+        next if _expected_warning_message(message, 'Unable to find sql file at')
```

Comment:
What's happening here? Why does it only occur for project_national?

Reply:
The testing project is run in debug mode. https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/measure.rb#L2415.

Perhaps this is an OSW (resstock) vs apply_measures (os-hpxml) thing? https://github.com/NREL/OpenStudio/blob/develop/src/workflow/RunEnergyPlus.cpp#L299.